tests/integration/test_s3.py

```diff
@@ -446,40 +446,45 @@ def test_s3_object_expiry(self):
         self._delete_bucket(bucket_name, [object_key])
 
     def test_s3_presigned_url_expired(self):
-        bucket_name = "test-bucket-%s" % short_uid()
-        client = self._get_test_client()
-        client.create_bucket(Bucket=bucket_name)
+        skip_before = config.S3_SKIP_SIGNATURE_VALIDATION
+        config.S3_SKIP_SIGNATURE_VALIDATION = False
```

Comment:
this can be rewritten with `patch.object`
https://github.com/localstack/localstack/blob/6f9e9fbb906dcd4704d6fab728d3f8c8256e1e56/tests/integration/test_s3.py#L1898-L1898

Reply:
Made adjustments in a separate PR https://github.com/localstack/localstack/pull/5026